<launch>
    <node name="usb_cam" pkg="usb_cam" type="usb_cam_node" output="log" >
        <param name="video_device" value="/dev/video0" />
	<remap from="/usb_cam" to="/vision/down_cam"/>
    </node>
</launch>
